canonical-web-and-design/canonicalwebteam.blog
# Use image_template for image processing

Enable cloudinary again for our blog articles.

This can be turned on or off with the flag `use_image_template=False` during the blog initialisation. 

QA
--
Browse to a blog article page that contains an image fx:
https://ubuntu-com-7940.demos.haus/blog/dell-xps-13-developer-edition-with-ubuntu-20-04-lts-pre-installed-is-now-available

or

https://ubuntu-com-7940.demos.haus/blog/design-and-web-team-summary-20th-july-2020

Check out the image url, it will be converted to a cloudinary url.

## Fixes #139
